dev-libs/beecrypt/beecrypt-4.2.1.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A strong and fast cryptography toolkit"
+DESCRIPTION="BeeCrypt is an ongoing project to provide a strong and fast \
+cryptography toolkit. It includes entropy sources, random generators, block \
+ciphers, hash functions, message authentication codes, multiprecision integer \
+routines, and public key primitives."
+HOMEPAGE="http://beecrypt.sourceforge.net/"
+COPYRIGHT="2009 Bob Deblier"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/beecrypt/beecrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="286f1f56080d1a6b1d024003a5fa2158f4ff82cae0c6829d3c476a4b5898c55d"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	beecrypt$secondaryArchSuffix = $portVersion
+	lib:libbeecrypt$secondaryArchSuffix = 7.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	beecrypt${secondaryArchSuffix}_devel = $portVersion
+	devel:libbeecrypt$secondaryArchSuffix = 7.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	beecrypt$secondaryArchSuffix == $portVersion base
+	devel:libgomp$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:diff
+	cmd:gcc$secondaryArchSuffix
```

Comment:
If I enable C++ in configure, the compilation fails with an error. It might be something with the ICU library which it needs or it might be something else, I haven't looked into it too much.